### DISCUSSION TITLE
Run script every N minutes

### DISCUSSION THREAD

**Discussion by omnigenous** [davmac314/dinit#278] [Q&A] [answered]
Another noob question from me, I'm on Artix if that matters.

I would like to run my custom bash script every 10 minutes right after boot. It's a simple script that runs a command and writes a file with command's output. Couldn't find how to do it all I see is systemd depended, but it's probably me not knowing what to search for.

I already installed `cronie` package and I know cron tab schedule I want is `*/10 * * * *` but not sure what to do next.

**Comment by davmac314** [comment]
A good start would probably be to read the cronie documentation, `man 5 crontab` would be enough.

I don't think this is really anything to do with Dinit?

**Comment by omnigenous** [comment, marked as answer]
Had to install `cronie-dinit` and run `sudo dinitctl start cronie`

**Reply by davmac314** [reply]
Ah right, I guess I didn't what you were asking. You probably want `sudo dinitctl enable cronie` if you need it to start every time you boot. The manual page is worth reading. The way Artix works generally, if you ever install a package providing a service, you will need to start it yourself (via `dinitctl enable` or at least `dinitctl start`).
